packages/devtools/test/integration_tests/app_test.dart

```diff
@@ -7,47 +7,52 @@ import 'package:test/test.dart';
 import '../support/cli_test_driver.dart';
 import 'integration.dart';
 
-void appTests() {
-  CliAppFixture appFixture;
-  BrowserTabInstance tabInstance;
-
-  setUp(() async {
-    appFixture = await CliAppFixture.create('test/fixtures/logging_app.dart');
-    tabInstance = await browserManager.createNewTab();
-  });
-
-  tearDown(() async {
-    await tabInstance?.close();
-    await appFixture?.teardown();
-  });
-
-  test('can switch pages', () async {
-    final DevtoolsManager tools =
-        DevtoolsManager(tabInstance, webdevFixture.baseUri);
-    await tools.start(appFixture);
-    await tools.switchPage('logs');
-
-    final String currentPageId = await tools.currentPageId();
-    expect(currentPageId, 'logs');
-  });
-
-  test('connect dialog displays', () async {
-    // start with no port
-    final Uri baseAppUri = webdevFixture.baseUri.resolve('index.html');
-    final DevtoolsManager tools =
-        DevtoolsManager(tabInstance, webdevFixture.baseUri);
-    await tools.start(appFixture, overrideUri: baseAppUri);
-
-    final ConnectDialogManager connectDialog = ConnectDialogManager(tools);
-
-    // make sure the connect dialog displays
-    await waitFor(() async => await connectDialog.isVisible());
-
-    // have it connect to a port
-    await connectDialog.connectTo(appFixture.servicePort);
-
-    // make sure the connect dialog becomes hidden
-    await waitFor(() async => !(await connectDialog.isVisible()));
+void main() {
+  group('app', () {
```

Comment:
what is the benefit of adding this group?

Reply:
I figured it'd keep the test names as before with a prefix - though it just occurred to me that you see the filename too, so probably it's superfluous now - I can remove :-)